lib/Reporters/SourceManager.cpp

```diff
@@ -12,7 +12,7 @@ SourceManager::~SourceManager() {
 
 std::string SourceManager::getLine(const SourceLocation &location) {
   LineOffset &lineOffset = getLineOffset(location);
-  assert(location.line < lineOffset.offsets.size());
+  assert(location.line <= lineOffset.offsets.size());
```

Comment:
Could you elaborate on this? Accessing `size()` element would be out of bounds, right?

Reply:
As discussed in person, the issue was related to the fact that the code didn't support getting the information for last lines in files.